substrate/frame/metadata-hash-extension/Cargo.toml

```
@@ -25,7 +25,7 @@ substrate-test-runtime-client = { workspace = true }
 sp-api = { workspace = true, default-features = true }
 sp-transaction-pool = { workspace = true, default-features = true }
 merkleized-metadata = { workspace = true }
-frame-metadata = { features = ["current"], workspace = true, default-features = true }
+frame-metadata = { features = ["current","unstable"], workspace = true, default-features = true }
```

I  think  `"unstable"`  will activate both "current" and "unstable IIRC

```suggestion
frame-metadata = { features = ["current", "unstable"], workspace = true, default-features = true }
```